proc/proc.go

```
@@ -408,6 +408,20 @@ func (dbp *Process) SwitchThread(tid int) error {
 	return fmt.Errorf("thread %d does not exist", tid)
 }
 
+// Change from current thread to the thread running the specified goroutine
+func (dbp *Process) SwitchGoroutine(gid int) error {
+	for i := range dbp.Threads {
+		if dbp.Threads[i].blocked() {
+			continue
+		}
+		g, _ := dbp.Threads[i].GetG()
```

Shouldn't ignore error here, we could panic if `g` is nil.